### PR TITLE
features section completed for readme update.

### PR DESCRIPTION
**Pull request by jared-esri**
@ldanzinger Ready for review. Thanks!

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/Features/FeatureLayerFeatureService/README.md [hunk 1]
```diff
@@ -1,16 +1,31 @@
...
+
+## How to use the sample
+
+Run the sample and view the feature service as an operational layer on top of the basemap. Zoom and pan around the map to see the features in greater detail.
+
 ## How it works
 
-The sample creates an instance of ServiceFeatureTable by supplying the URL to the REST endpoint of the feature service. The FeatureLayer is then supplied with the ServiceFeatureTable and added to the Map.
+1. Create a `ServiceFeatureTable` from a URL.
+1. Create a feature layer from the service feature table.
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/Features/FeatureLayerFeatureService/README.md]: numbering off

File: ArcGISRuntimeSDKQt_CppSamples/Features/FeatureLayerFeatureService/README.md [hunk 2]
```diff
@@ -1,16 +1,31 @@
...
+1. Add the feature layer to your ArcGISMap using `ArcGISMap::operationalLayers()::append(FeatureLayer)`.
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/Features/FeatureLayerFeatureService/README.md]: ArcGISMap -> Map

File: ArcGISRuntimeSDKQt_CppSamples/Features/FeatureLayerGeodatabase/README.md [hunk 3]
```diff
@@ -1,24 +1,52 @@
...
+
+The sample shows trailheads in the greater Los Angeles area displayed on top of a vector tile basemap.
+
+## Additional information
+
+One of the ArcGIS Runtime data set types that can be accessed via the local storage of the device (i.e. hard drive, flash drive, micro SD card, USB stick, etc.) is a mobile geodatabase. A mobile geodatabase can be provisioned for use in an ArcGIS Runtime application by ArcMap. The following provide some helpful tips on how to create a mobile geodatabase file:
+
+In ArcMap, choose File > Share As > ArcGIS Runtime Content from the menu items to create the .geodatabase file (see the document: http://desktop.arcgis.com/en/arcmap/latest/map/working-with-arcmap/creating-arcgis-runtime-content.htm).
+
+Note: You could also use the 'Services Pattern' and access the Geodatabase class via a Feature Service served up via ArcGIS Online or ArcGIS Enterprise. Instead of using the Geodatabase class to access the .geodatabase file on disk, you would use GeodatabaseSyncTask point to a Uri instead. For more information review the document: https://developers.arcgis.com/qt/latest/cpp/guide/offline.htm.
```
Comment (ldanzinger) [inline ArcGISRuntimeSDKQt_CppSamples/Features/FeatureLayerGeodatabase/README.md]: I've seen a few references to URI. We should change this to URL throughout - we may want to add this to our list of things to change. I've not marked all of these up in the PR

**Comment by jared-esri** [comment]
@ldanzinger should be good to go.

**Pull request merged**